css/index.css

```diff
@@ -328,6 +325,26 @@ header .logo {
   padding-right: 20px;
 }
 
+@media (min-width: 769px) {
```

Comment:
Impressed with your implementation with min-width! You're definitely ahead of the curve here and hope you continue to do so!